src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -538,8 +539,9 @@ struct DeviceVhostUserFsPci {
 }
 
 impl DeviceVhostUserFsPci {
-    fn new(chardev: &str, tag: &str) -> DeviceVhostUserFsPci {
+    fn new(chardev: &str, tag: &str, bus_type: String) -> DeviceVhostUserFsPci {
```

Comment:
Because you add the bus type as argument, I would suggest to rename this implementation from `DeviceVhostUserFsPci` to impl `DeviceVhostUserFs`.

Reply:
Thanks for pointing it out. I will rename it.